tox.ini

```diff
@@ -106,3 +106,15 @@ deps =
     {[testenv]deps}
     reppy
     robotexclusionrulesparser
+
+[testenv:py35-asyncio]
+basepython = python3.5
+deps = {[testenv]deps}
+commands =
+    py.test --cov=scrapy --cov-report= --reactor=asyncio {posargs:scrapy tests}
+
+[testenv:py38-asyncio]
+basepython = python3.8
+deps = {[testenv]deps}
+commands =
+    py.test --cov=scrapy --cov-report= --reactor=asyncio {posargs:scrapy tests}
```

Comment:
:lipstick: I _think_ you can omit the `deps` line in these two environments.

You could also define `[asyncio]` with this `commands` and then use `commands = {[asyncio]commands}` in these two environments. Although I’m not sure it’s worth it for only 2 entries, so up to you.